Fixes #71. Refactored code a little.

mbykovskyy opened the pull request:
Forgot how to link pull request to an issue. :smile: Fixes #71.

marcbachmann commented:
Thanks. I finally had some time to have a look at this.
I tried to extend the js example file: https://github.com/marcbachmann/node-html-pdf/commit/396a2522cb7a24aca1560f52c70bfae0aea80e63
Now the page throws an error :cry: . phantoms `page.onError` reports that jQuery is not defined.
Did it work for you?

mbykovskyy commented:
@marcbachmann I figured out why it wasn't working. I saw this problem before. Resources doesn't get fetched over HTTPS. It's a `phantom` bug and I think I saw an open ticket for it in their repo. I changed the link to HTTP and also added your change.

marcbachmann commented:
Oh, you're right. It also works if you add the `ignore-https-errors` option. So it works like this:
`pdf.create(enrichedHtml, phantomArgs: ['--ignore-ssl-errors=true'])`

marcbachmann commented:
Damn, I just merged that in the command line. Apparently git isn't able to track remote repositories. Or it might be possible because I only merged the first commit. Anyways, this is now in master and released as version 1.4.0. Thanks
